airflow/api_fastapi/execution_api/routes/variables.py

```diff
@@ -67,7 +67,28 @@ def get_variable(variable_key: str, token: deps.TokenDep) -> VariableResponse:
     return VariableResponse(key=variable_key, value=variable_value)
 
 
-def has_variable_access(variable_key: str, token: TIToken) -> bool:
+@router.put(
+    "/{variable_key}",
+    responses={
+        status.HTTP_401_UNAUTHORIZED: {"description": "Unauthorized"},
+        status.HTTP_403_FORBIDDEN: {"description": "Task does not have access to the variable"},
+    },
+)
+def post_variable(variable_key: str, body: VariablePostBody, token: deps.TokenDep):
```

Comment:
```suggestion
def put_variable(variable_key: str, body: VariablePostBody, token: deps.TokenDep):
```

or just `variable_set` ?

Reply:
`put_variable` sounds better imo. Adding for now.